### ISSUE TITLE
Error cannot open ext/assimp when running the build file.

### ISSUE THREAD

**Issue by PsychoCoffee** [pkurth/D3D12Renderer#3]
When double clicking on the generate2022.bat file, it gives me the following error:

Error: cannot open ext/assimp: No such file or directory

When going into the assimp folder, it seems to be empty.
![image](https://user-images.githubusercontent.com/89361918/212553763-f4a34767-3041-49be-842c-52e2e2e45fe0.png)

**Comment by PsychoCoffee** [comment]
Fixed it by going to the ext folder in the repository, and downloading the yaml and assimp folder seperately, then extracting them and putting them in their directory in the downloaded project.

**Comment by pkurth** [comment]
Thanks for checking out my project. Did the other dependencies work right away? Assimp and Yaml should be cloned automatically if you cloned the project recursively.

**Comment by PsychoCoffee** [comment]
Hey! I am impressed by this project and I wish you the best.

I had mostly issue with the Assimp, Yaml and ImGui dependencies, as they didn't clone whilst downloading.
I'm not sure if I downloaded the project the right way, all I did was to click on the code, and then download button, which caused those folders to become empty.

I did try disabling anti-virus before downloading as well but that didn't fix the issue
